.fixtures.yml

```diff
@@ -6,5 +6,12 @@ fixtures:
     common:
       repo: 'https://github.com/ghoneycutt/puppet-module-common.git'
       ref: 'v1.5.0'
+    apache:
+      repo: 'https://github.com/puppetlabs/puppetlabs-apache.git'
+      ref: '0.6.0'
```

Comment:
0.6.0 is the 'newest' that have apache::vhost::template available. This param was deprecated with 0.7.0. To get this module to use the latest version it first needs to get refactored. Let me finish the spec tests before changing functionality.